applications/ray/kuberay-tpu-webhook/README.md

```diff
@@ -46,8 +46,15 @@ accelerator_type       = "nvidia-tesla-t4"
 1. `make install-cert-manager` - it may take up to two minutes for the certificate to become ready
 
 2. If deploying webhook across multiple namespaces: `make install-reflector`
```

Comment:
Could you add a comment in the Makefile to explain what `install-reflector` does?

Reply:
^ follow-up question, why do we need reflector installed? In what scenarios do we need the webhook installed on multiple namespaces?

Reply:
After thinking about it more I'm just going to remove the reflector, I used it initially when I had the webhook scoped to one namespace and deployed it in a different one than the TLS certificate, I don't think there's much of a use case for it with the current webhook.